examples/MANN/MANNAutoregressive.ipynb

```diff
@@ -0,0 +1,237 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the path of the robot model\n",
+    "robot_model_path = str(resolve_robotics_uri_py.resolve_robotics_uri(\"package://ergoCub/robots/ergoCubSN001/model.urdf\"))\n",
+    "ml = idyn.ModelLoader()\n",
+    "ml.loadReducedModelFromFile(robot_model_path, params_network.get_parameter_vector_string(\"joints_list\"))\n",
+    "viz = MeshcatVisualizer()\n",
+    "viz.load_model(ml.model())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Network initialization\n",
+    "In this section we initialize the network. In particular we first instantiate the network and then we load the parameters. The parameters are loaded from the `config_mann.toml` file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the trajectory generator\n",
+    "mann_trajectory_generator = blf.ml.MANNAutoregressive()\n",
+    "assert mann_trajectory_generator.set_robot_model(ml.model())\n",
+    "assert mann_trajectory_generator.initialize(params_network)\n",
+    "\n",
+    "# Create the input builder\n",
+    "input_builder = blf.ml.MANNAutoregressiveInputBuilder()\n",
+    "assert input_builder.initialize(params_joypad)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to reset the network to the initial state. In particular, we need to reset  the joint state and the base pose in a given configuration already seen by the network during traning."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initial joint positions configuration. The serialization is specified in the config file\n",
+    "joint_positions = np.array([-0.10922017141063572, 0.05081325960010118, 0.06581966291990003, -0.0898053099824925, -0.09324922528169599, -0.05110058859172172,\n",
+    "                            -0.11021232812838086, 0.054291515925228385,0.0735575862560208, -0.09509332143185895, -0.09833823347493076, -0.05367281245082792,\n",
+    "                            +0.1531558711397399, -0.001030634273454133, 0.0006584764419034815,\n",
+    "                            -0.0016821925351926288, -0.004284529460797688, 0.030389771690123243,\n",
+    "                            -0.040592118429752494, -0.1695472679986807, -0.20799422095574033, 0.045397975984119654,\n",
+    "                            -0.03946672931050908, -0.16795588539580256, -0.20911090583076936, 0.0419854257806720])\n",
+    "\n",
+    "# Initial base pose. This pose makes the robot stand on the ground with the feet flat\n",
+    "initial_base_height = 0.7748\n",
+    "base_pose = manif.SE3.Identity()\n",
+    "quat = np.array([ 0, -0.0399893, 0, 0.9992001])\n",
+    "quat = quat / np.linalg.norm(quat)\n",
+    "base_pose = manif.SE3([0, 0, initial_base_height], quat)"
```

Comment:
A possible suggestion could be to move these to a configuration file

Reply:
Removed here: https://github.com/ami-iit/bipedal-locomotion-framework/pull/771/commits/7d8b7a3aaf5e392e6441814f30fbf8a3139cc898